statistics.md

```diff
@@ -119,59 +119,172 @@ Before v5.3.0, TiDB uses the reservoir sampling method to collect statistics. Si
+
+    2. After the query pattern of your business is stable, collect the statistics of `PREDICATE COLUMNS` by using the following syntax:
 
-+ To collect statistics of partition in all `PartitionNameList`s in `TableName`:
+        {{< copyable "sql" >}}
+
+        ```sql
+        ANALYZE TABLE TableName PREDICATE COLUMNS [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+        ```
+
+        Besides collecting statistics for the `PREDICATE COLUMNS` in a specified table, this syntax collects statistics for the indexed columns and all indexes in that table at the same time.
+
+        > **Note:**
+        >
+        > - If the `mysql.column_stats_usage` system table does not contain any `PREDICATE COLUMNS` record for that table, executing the above statement collects statistics for all columns and all indexes in that table.
+        > - After using this syntax to collect statistics, when executing a new type of SQL query, the optimizer might temporarily use the old or pseudo column statistics and will collect the new column statistics from the next time.
+
+- To collect statistics for all columns and all indexes, use the following syntax:
 
     {{< copyable "sql" >}}
 
     ```sql
-    ANALYZE TABLE TableName PARTITION PartitionNameList [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
+    ANALYZE TABLE TableName ALL COLUMNS [WITH NUM BUCKETS|TOPN|CMSKETCH DEPTH|CMSKETCH WIDTH]|[WITH NUM SAMPLES|WITH FLOATNUM SAMPLERATE];
     ```
 
-+ To collect statistics of some columns for the partitions in all `PartitionNameList`s in `TableName`:
+If you want to persist the column configuration in the `ANALYZE` statement (including `COLUMNS ColumnNameList`, `PREDICATE COLUMNS`, and `ALL COLUMNS`), set the value of the `tidb_persist_analyze_options` system variable to `ON` to enable the [ANALYZE configuration persistence](/statistics.md#analyze-configuration-persistence) feature. After enabling the ANALYZE configuration persistence feature:
+
+- When TiDB collects statistics automatically or when you manually collect statistics by executing the `ANALYZE` statement without specifying the column configuration, TiDB continues using the previously persisted configuration for statistics collection.
+- When you manually execute the `ANALYZE` statement multiple times and specify the column configuration each time, TiDB overwrites the previously recorded persistent configuration using the new configuration specified by the latest `ANALYZE` statement.
+
+If you want to see which columns in a table are `PREDICATE COLUMNS` and the statistics for which columns have been collected, you can use the following syntax.
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW COLUMN_STATS_USAGE [ShowLikeOrWhere];
+```
+
+The `SHOW COLUMN_STATS_USAGE` statement returns the following 6 columns:
+
+| Column name | Description            |
```

Comment:
It should be "Column name here". The Chinese version here is incorrect. @xuyifangreeneyes PTAL.

Reply:
I check other `SHOW`s and they also use 语法元素 in Chinese and Syntax Element in English.

Reply:
```suggestion
| Syntax Element | Description            |
```

Reply:
I don't think the "语法元素“ in Chinese is correct because "语法元素“ or "syntax" is used for command input, while we are now introducing the column names in the output of the `Show` statement.